### DISCUSSION TITLE
X-lab 社区论坛来了

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#31] [1️⃣ 新鲜事]
👋，GitHub org 层面的论坛功能，比较适合作为 X-lab 整体层面事务的讨论，可以好好用起来~ 🎉

<div align=center>
<img src="https://user-images.githubusercontent.com/15010826/166107957-db197921-57b9-467b-a100-186b87a3d65a.png" width="650px">
</div>

**Comment by will-ww** [comment]
GitHub 的组织论坛设定，需要依托一个具体的 repo 的，不用新建一个，用一个已有的即可，这里选择 OpenWonderland 应该是最合适的，即下面两个链接是一样的：

- https://github.com/orgs/X-lab2017/discussions
- https://github.com/X-lab2017/open-wonderland/discussions

Enjoy~

![image](https://user-images.githubusercontent.com/15010826/166110443-ccd4e533-5b8e-495a-8525-6a91ef8b1840.png)

**Reply by tyn1998** [reply]
这个feature好周到！还有一个发现：org下的discussions默认以卡片视图呈现，也能选择列表视图；repo下的discussions只有列表形式~

**Reply by will-ww** [reply]
很仔细，OK，感觉做了用户体验上的一些优化~
